cvat-core/src/api-implementation.js

```diff
@@ -184,8 +190,12 @@
                 }
             }
 
+            if ('projectId' in filter && Object.keys(filter).length > 1) {
+                throw new ArgumentError('Do not use the filter field "projectId" with other');
+            }
```

Comment:
Couldn't this field be used with ``page``?

Reply:
In general you are right, it was added in case of usage for only getting all project tasks.